piezo_web_app/build_package.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [ -d ./dist ]; then
+  echo "Removing existing dist folder"
+  rm -rf ./dist
+fi
+
+python setup.py sdist bdist_wheel
```

Comment:
As below again, we likely will never be building the app as a package as we will be deploying it in a pod